homeassistant/components/pglab/entity.py

```diff
@@ -0,0 +1,59 @@
+"""Entity for PG LAB Electronics."""
+
+from __future__ import annotations
+
+from pypglab.device import Device
+from pypglab.entity import Entity
+
+from homeassistant.core import callback
+from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC, DeviceInfo
+from homeassistant.helpers.entity import Entity as HA_Entity
+
+from .const import DEVICE_ALREADY_DISCOVERED, DOMAIN
+
+
+class BaseEntity(HA_Entity):
+    """Representation of a PGLAB entity in Home Assistant."""
+
+    def __init__(self, platform: str, device: Device, entity: Entity) -> None:
```

Comment:
Why do we need the platform?

Reply:
I believe it's used for the second part of this integration (cover and sensor). For this patch is not important. I'm going to remove it. Thank you for spotting it.